src/App.jsx

```diff
@@ -1,30 +1,22 @@
 import React from 'react';
-import { ApplicationInsights, SeverityLevel } from '@microsoft/applicationinsights-web';
+import { SeverityLevel } from '@microsoft/applicationinsights-web';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```

Comment:
```
./src/App.jsx
  Line 3:  'Router' is defined but never used  no-unused-vars
```

Reply:
I created a bug related to failing tests, will take care of it later this iteration